app/models/date_range.rb

```diff
@@ -1,7 +1,9 @@
 class DateRange
   # Step 1: only in use with Conference date range.
-  # Validated in Conference: presence and chronological order
-  #  because the input is done there.
+  # Validation is supposed to be in input (e.g. Conference.rb), but
+  # with the new Importer for adding conferences, the presence validation
+  # of the dates is disabled. Validation of chronological order is still present.
+  # Importing a Conference without dates will log an error in the imports_logger file
```

Comment:
I would remove this line (reference to implementation detail in another file/class)